modules/tinymce/CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Fixed
-- Typing after deleting formatted content could remove a space at the start of the typing. TINY-9310
+- Typing after deleting formatted content could remove a space at the start of the typing. #TINY-9310
+- Disable inserting an accordion into noneditable elements. #TINY-9760
```

Comment:
It seems not necessary to add Fixed change log entries for unreleased new plugin

Reply:
Correct, if the bug was never released we don't add a changelog entry for it

```suggestion
```